src/review/ios/spider.js

```diff
@@ -1,9 +1,34 @@
 const rp = require('request-promise')
+const { PRESET_REGIONS } = require('../common')
 
-const getReviews = async (id) => {
-  const body = await rp(`https://itunes.apple.com/us/rss/customerreviews/page=1/sortBy=mostRecent/id=${id}/json`)
+const getReviewsByRegion = async (id, region) => {
+  const body = await rp(
+    `https://itunes.apple.com/${region}/rss/customerreviews/page=1/sortBy=mostRecent/id=${id}/json`
+  )
   const json = JSON.parse(body)
   return json.feed.entry
 }
-
-module.exports = { getReviews }
+const getReviews = id => {
+  let promises = Object.keys(PRESET_REGIONS).map(index =>
+    getReviewsByRegion(id, PRESET_REGIONS[index])
+  )
+  return Promise.all(promises).then(rawArr => {
+    let arr = []
+    for (let i in rawArr) {
+      rawArr[i] && arr.push(rawArr[i])
+    }
+    return arr
+  })
+}
+const getFlattenReviews = async id => {
+  let collections = await getReviews(id)
```

Comment:
If an app ID can only exist in one region, `collections` is a single element array. Maybe we can improve the code a little bit.